tasks/main.yml

```diff
@@ -10,14 +10,27 @@
   with_items:
     - /opt/cadvisor
 
-- name: cadvisor | download
-  become: true
-  get_url:
-    url: "{{ cadvisor_download }}"
-    checksum: "sha256:{{ cadvisor_checksum }}"
+- block:
```

Comment:
What is the rationale for using a `block` as it only wraps one tasks (as opposed to attaching the `when` clause to `get_url` as for the `copy` task below?

Reply:
You're right, there's no point in it being there. I think it's left over from another feature I think about. I'm going to remove it, right now. Thank for the comment